mobile/lib/modules/map/widgets/campaign_item.widget.dart

```diff
@@ -92,7 +92,7 @@ class CampaignItem extends StatelessWidget {
                     ],
                   ),
                   Text(
-                    currentCampaign.organizationName,
+                    currentCampaign.organizationName!,
```

Comment:
I recommend using null check and add default value for null safety `currentCampaign.organizationName ?? 'Some foo default value'`